olm/quickstarts/rhosak-openshift-connect-quickstart.yaml

```diff
@@ -3,160 +3,328 @@ kind: ConsoleQuickStart
+
+            ii. Select the project that you previously created.
 
-        The Red Hat OpenShift Application Services operator allows you to represent Red Hat Cloud Services as first class citizens in your OpenShift environment.
-        This enables you to work and integrate with these services using standard OpenShift/Kubernetes features and APIs.
+        1. If you don't already have a command-line window open in the pod for your tools application, open one, as described earlier in this quick start.
 
-        We will use the `rhoas` CLI to connect your Kafka instance to your OpenShift project. The `rhoas` CLI uses a browser-based login flow by default
-        (meaning, when you execute `rhoas login` it will start a sign-in flow in your web browser). Since our terminal in OpenShift does not have access to a browser,
-        we need to login using a token.
+        1. Log in to the OpenShift CLI using a token.
 
-        1. Navigate to [https://cloud.redhat.com/openshift/token](https://cloud.redhat.com/openshift/token) and copy the token to your clipboard using the **Copy to clipboard** button.
+            i. In the upper-right corner of the console, next to your user name, click the drop-down menu. Select **Copy login command**.
+               A new page opens.
 
-        1. In your terminal, execute the RHOAS CLI login command, replacing *{token}* with the token you've just copied to your clipboard: `rhoas login --token={token}`.
-        You should see a response saying: **You are now logged in as {user}**.
+            ii. Click the **Display Token** link.
 
-        1. List your Kafka instances by executing the command: `rhoas kafka list`. You should see a Streams for Apache Kafka instance that has been provisioned for you.
+            iii. In the section entitled **Log in with this token**, copy the full `oc login` command shown.
 
-        The **rhoas** CLI uses the **oc** CLI to determine the OpenShift instance, and project, that the Streams for Apache Kafka instance should be connected to.
-        Let's verify that your **oc** client is properly connected to your OpenShift project.
+            v. On the command line, paste the login command you copied. Right-click on the command line and select **Paste**. An example is shown below.
 
-        1. In your terminal, first login the **oc** client to our OpenShift instance. To do this:
-            1. Click on your username in the top-right of the OpenShift console and click **Copy login command**
-            1. A new tab opens in your browser with the OpenShift Developer Sandbox login screen. In the *Log in with...* box, select **DevSandbox**.
-            1. Click on **Display Token**.
-            1. Copy the `oc login` command.
-            1. Go to your terminal, and execute the command you just copied.
+            ```
+            $ oc login --token=sha256~0WJOGcha7EOAkCmJpSSb6pyo2EawSUwZJKDEw3c-Ox4 --server=https://example.com:6443
+            ```
```

Comment:
I notice that none of the code snippets have labels, which might be a limitation of the YAML? I thought I remembered you used labels for all snippets in your other draft. Where possible, I think the standing recommendation for optimized usability and readability is to include labels for images, tables, diagrams, and code snippets. It's a huge help to readers buzzing through steps especially.

So here would be something like:

````
... the following example: 

**Example OpenShift login command**
```
$ oc login --token=sha256~0WJOGcha7EOAkCmJpSSb6pyo2EawSUwZJKDEw3c-Ox4 --server=https://example.com:6443
```
````

So I know it would be annoying, but if this is possible in the YAML, I'd recommend a label for all code snippets throughout. Feel free to disregard if no time and maybe come back another time. This is the direction of our other content and is a huge user payoff for just a little additional effort for writers, IMO.

Reply:
Yep, @sterobin  - per our offline discussion, this **is** a limitation of the YAML-based `ConsoleQuickStart` CR. For now a line of bolded body text is our best bet, as you've done above. https://github.com/patternfly/patternfly-quickstarts/issues/11 has been created to add support for captions on code blocks in OpenShift quick starts.

I'll be adding a line of bolded text to each code block.